### PR TITLE
Preparing for 1.4

### PR DESCRIPTION
**Pull request by zalun**
Before we will push it live we need to have ABH 1.4 downloadable on AMO
- display a warning if ABH is not responding with an object containing `then`
- set the desired ABH number to 1.4

### CODE REVIEW

File: media/base/js/flightdeck/FlightDeck.js [hunk 1]
```diff
@@ -254,24 +253,26 @@ module.exports = new Class({
...
-                            } else {
-                                if (result) {
-									log.debug('Unsucessful result:', result);
-								}
-                                this.warning.alert(
-                                    'Add-on Builder',
-                                    'Wrong response from Add-on Builder Helper. Please <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=573778">let us know</a>'
-                                );
-                            }
+                            var result = ABH().send("install", responseText).then(function(response) {
```
Comment (seanmonstar) [inline media/base/js/flightdeck/FlightDeck.js]: it's async now, so `result` variable isn't needed

File: media/base/js/flightdeck/FlightDeck.js [hunk 2]
```diff
@@ -294,12 +295,15 @@ module.exports = new Class({
...
-        log.info('uninstalling ' + jid);
-        var result = ABH().send({cmd:'uninstall'});
-        if (result.success) {
-			this.fireEvent('xpi_uninstalled');
-		}
+    uninstallXPI: function() {
+        log.debug('uninstalling add-on');
+        ABH().send('uninstall').then(function(response){
+            if (response.success) {
+                // XXX: no idea why it's not working without this line
```
Comment (seanmonstar) [inline media/base/js/flightdeck/FlightDeck.js]: really? tried removing it, again? we should find out why... and if not, want to log a useful message there?

**Comment by seanmonstar** [comment]
`mozFlightDeck` is also accessed in PackageController, line 200, to toggle the debug console.

File: media/base/js/flightdeck/FlightDeck.js (hunk 2 above)
Comment (zalun) [inline media/base/js/flightdeck/FlightDeck.js]: the message is later in whenXpiUninstalled
I was adding and removing this line a few times as I couldn't believe it myself
I also restarted firefox althought not entire system :)

File: apps/jetpack/management/commands/add_core_lib.py [hunk 3]
```diff
@@ -10,11 +10,17 @@ class Command(BaseCommand):
...
+                help="Apply this options to ``cfx xpi`` command"),
+            make_option('--useversion',
+                dest="version",
+                default=None,
+                help="Version string to show in Builder"),)
 
     def handle(self, sdk_dir_name, *args, **kwargs):
         #try:
-        create_or_update_SDK(sdk_dir_name, options=kwargs['options'])
+        create_or_update_SDK(sdk_dir_name,
```
Comment (seanmonstar) [inline apps/jetpack/management/commands/add_core_lib.py]: you wouldnt rather change the method header?

`def handle(self, sdk_dir_name, version=None, options=None, *args, **kwargs):`

**Comment by seanmonstar** [comment]
the comment about `handle` is more just a question. so r+!

**Pull request merged**